Week-2/Assignment_2_1/styles/style.css

```diff
@@ -0,0 +1,47 @@
+* {
+  box-sizing: border-box;
+}
+
+header {
+  margin-top: 5%;
+  text-align: center;
+}
+
+h1,
+h2 {
+  font-weight: 400;
+  font-family: DauphinPlain;
+}
+h3,
+h4,
+h5 {
+  font-weight: 400;
+}
+
+#section {
```

Comment:
lets use class instead of id

Reply:
below as well

Reply:
Replace id with class
![image](https://user-images.githubusercontent.com/108617696/182028107-861e2c49-a22c-4c82-97de-e7e3aed8c6a2.png)